core/src/main/java/com/es/core/service/impl/PlacedOrdersService.java

```diff
@@ -0,0 +1,23 @@
+package com.es.core.service.impl;
+
+import com.es.core.model.order.Order;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class PlacedOrdersService {
```

Comment:
No, bad approach to organize bean like this, it's part of orderService, just add methods here.